include/logger/logger.hpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LOGGER_HPP_
+#define LOGGER_HPP_
+
+#include <mutex>
+
+#include <aos/common/tools/log.hpp>
+
+/**
+ * Logger instance.
+ */
+class Logger {
+public:
+    /**
+     * Log backends.
+     */
+    enum class Backend {
+        eStdIO,
+        eJournald,
+    };
+
+    /**
+     * Initializes logging system.
+     *
+     * @return aos::Error.
+     */
+    aos::Error Init();
+
+    /**
+     * Sets logger backend.
+     *
+     * @param backend logger backend.
+     */
+    void SetBackend(Backend backend)
+    {
+        std::lock_guard lock(sMutex);
+
+        sBackend = backend;
+    }
+
+    /**
+     * Sets current log level.
+     *
+     * @param level log level.
+     */
+    void SetLogLevel(aos::LogLevel level)
+    {
+        std::lock_guard lock(sMutex);
+
+        sLogLevel = level;
+    }
+
+private:
+    static constexpr auto cColorTime    = "\033[90m";
+    static constexpr auto cColorDebug   = "\033[37m";
+    static constexpr auto cColorInfo    = "\033[32m";
+    static constexpr auto cColorWarning = "\033[31m";
```

Comment:
Could you set `"\033[33m";` ? Here is error in our case.

Reply:
Done